base/util_testing.go

```diff
@@ -568,7 +568,7 @@ var GlobalTestLoggingSet = AtomicBool{}
 
 // SetUpGlobalTestLogging sets a global log level at runtime by using the SG_TEST_LOG_LEVEL environment variable.
 // This global level overrides any tests that specify their own test log level with SetUpTestLogging.
-func SetUpGlobalTestLogging(ctx context.Context, m *testing.M) (teardownFn func()) {
+func SetUpGlobalTestLogging(ctx context.Context) (teardownFn func()) {
```

Comment:
Would prefer to keep `_ *testing.M` in the signature so it's not accidentally misused.

Reply:
The reason I removed this is that I call this function from `ResetGlobalTestLogging` which only has `* testing.T`. I could keep testing.M and pass nil in `ResetGlobalTestLogging` but I can't think of a great a way to convey this otherwise.